cypress/integration/navigateHomeAndContactsPages.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('navigateHomeAndContactsPages', function() {
+
+    const operations = [
+        {
+            // Estonian
+            homePage: 'https://ut2019.voog.com/et',
+            contactPage: 'https://ut2019.voog.com/contact-1'
+
+        },
+        {
+            // English
+            homePage: 'https://ut2019.voog.com/en',
+            contactPage: 'https://ut2019.voog.com/contact'
+        }
+    ];
+
+    operations.forEach((operation) => {
+        it('Navigated successfully', function() {
+            cy.visit(operation.homePage);
+
+            cy.visit(operation.contactPage);
```

Comment:
Here you test the site navigation only through direct url visits. The user can however navigate the site by clicking on the Contacts and Homepage menu links. Maybe they get broken, but direct url visits would work and your test keeps passing.